packages/babel-code-frame/test/index.js

```diff
@@ -94,53 +113,109 @@ describe("@babel/code-frame", function () {
     );
   });
 
-  test("opts.highlightCode", function () {
-    const rawLines = "console.log('babel')";
-    const result = codeFrame(rawLines, 1, 9, { highlightCode: true });
-    const stripped = stripAnsi(result);
-    expect(result.length).toBeGreaterThan(stripped.length);
-    expect(stripped).toEqual(
-      ["> 1 | console.log('babel')", "    |         ^"].join("\n"),
-    );
-  });
+  describe("when colors are supported", () => {
+    stubColorSupport(true);
 
-  test("opts.highlightCode with multiple columns and lines", function () {
-    // prettier-ignore
-    const rawLines = [
-      "function a(b, c) {",
-      "  return b + c;",
-      "}"
-    ].join("\n");
+    test("opts.highlightCode", function () {
+      const rawLines = "console.log('babel')";
+      const result = codeFrame(rawLines, 1, 9, { highlightCode: true });
+      const stripped = stripAnsi(result);
+      expect(result.length).toBeGreaterThan(stripped.length);
+      expect(stripped).toEqual(
+        ["> 1 | console.log('babel')", "    |         ^"].join("\n"),
+      );
+    });
 
-    const result = codeFrameColumns(
-      rawLines,
-      {
-        start: {
-          line: 1,
-          column: 1,
+    test("opts.highlightCode with multiple columns and lines", function () {
+      // prettier-ignore
+      const rawLines = [
+        "function a(b, c) {",
+        "  return b + c;",
+        "}"
+      ].join("\n");
+
+      const result = codeFrameColumns(
+        rawLines,
+        {
+          start: {
+            line: 1,
+            column: 1,
+          },
+          end: {
+            line: 3,
+            column: 1,
+          },
         },
-        end: {
-          line: 3,
-          column: 1,
+        {
+          highlightCode: true,
+          message: "Message about things",
         },
-      },
-      {
-        highlightCode: true,
-        message: "Message about things",
-      },
-    );
-    const stripped = stripAnsi(result);
-    expect(stripped).toEqual(
-      // prettier-ignore
-      [
-        "> 1 | function a(b, c) {",
-        "    | ^^^^^^^^^^^^^^^^^^",
-        "> 2 |   return b + c;",
-        "    | ^^^^^^^^^^^^^^^",
-        "> 3 | }",
-        "    | ^ Message about things",
-      ].join('\n'),
-    );
+      );
+      const stripped = stripAnsi(result);
+      expect(result.length).toBeGreaterThan(stripped.length);
```

Comment:
Added a new test here to assert that the `result` has been highlighted as expected from `{ highlightCode: true }`.